src/render_tree/text.rs

```diff
@@ -15,66 +15,41 @@ impl TextNode {
     /*
         NOTE: I got the default font sizes from https://stackoverflow.com/a/70720104
     */
-    pub fn new_heading1() -> Self {
+
+    fn new(fs: f64, bold: bool) -> Self {
         Self {
             value: "".to_owned(),
             font: DEFAULT_FONT.to_owned(),
-            font_size: 37.,
-            is_bold: true,
+            font_size: fs,
+            is_bold: bold,
         }
     }
 
+    pub fn new_heading1() -> Self {
+        TextNode::new(37., true)
+    }
+
     pub fn new_heading2() -> Self {
-        Self {
-            value: "".to_owned(),
-            font: DEFAULT_FONT.to_owned(),
-            font_size: 27.5,
-            is_bold: true,
-        }
+        TextNode::new(27.5, true)
     }
 
     pub fn new_heading3() -> Self {
-        Self {
-            value: "".to_owned(),
-            font: DEFAULT_FONT.to_owned(),
-            font_size: 21.5,
-            is_bold: true,
-        }
+        TextNode::new(21.5,true)
```

Comment:
formatter might complain about having no space between the commas (compared to the earlier `TextNode::new`.) It's minor but if we fix it here, we wouldn't have to fix it in my branch after merging

Reply:
Fixed. It's sometimes hard to see, RustRover and all the other Jetbrains IDEs have this "annotation" or something with the name before the value

Reply:
Cool, I am ok to merge this into my branch now to make sure that everything still passes and the rendering on the user agent still works. Thanks again!